.circleci/config.yml

```diff
@@ -24,6 +24,7 @@ workflows:
       - macos-build
       - format-check
       - header-check
+      - doc-gen-job
```

Comment:
This is only for test purposes and will be removed in subsequent update to this PR.